HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -78,6 +79,7 @@ public IActionResult GasBoiler_Post(GasBoilerViewModel viewModel)
     }
 
     [HttpGet("direct-to-eco/")]
+    [SessionExpiry]

```

Comment:
If it's simple, can we change this so that it applies to all methods on the controller except ones with a specific attribute?

Reply:
I'll have a go - I might be able to apply this at the controller level and add a second attribute to 'exclude' actions.